docs/planning/backend/app/api/recommendations.py

```
@@ -0,0 +1,275 @@
+  "patterns": ["observed patterns in their preferences"],
+  "occasions": ["when they seem to prefer wearing fragrance"],
+  "summary": "2-3 sentence summary of insights"
+}}
+"""
+
+    try:
+        result = openrouter.chat_json(
+            messages=[{"role": "user", "content": prompt}],
+            max_tokens=800,
+        )
+        return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)